src/RandomStatusMiddleware.php

```diff
@@ -96,6 +96,6 @@ static function (int $statusCode) use ($currentStatus) : bool {
             }
         );
 
-        return $response->withStatus($statusCodesToPickFrom[random_int(0, count($statusCodesToPickFrom) - 1)]);
+        return $response->withStatus(array_rand($statusCodesToPickFrom));
```

Comment:
Hmm, I'm hesitant to accept this, as:

1) I'm uncertain whether `array_rand` can generate crytographically secure random integers?
2) `array_rand` returns the key, not the value, so would have to be `$statusCodesToPickFrom[array_rand($statusCodesToPickFrom)]` or something, I think?

Reply:
The documentation mentions the following.
>It uses a pseudo random number generator that is not suitable for cryptographic purposes.

Reply:
Yeah, this definitely requires cryptographically secure random numbers...